### PR TITLE
Fix acks

### PR DESCRIPTION
**Pull request by agouin**
Use `map[string]string` for event attributes inside event since events can have many attributes

### CODE REVIEW

**Review by jackzampolin — APPROVED** [review]

**Comment by DavidNix** [comment]
In this PR or a followup, do we want to include an `ibctest/go.mod` which points to the latest of ibctest?

File: relayer/provider/cosmos/provider.go [hunk 1]
```diff
@@ -1153,55 +1153,53 @@ func (cc *CosmosProvider) relayPacketsFromResultTx(ctx context.Context, src, dst
...
+			case dstPortTag:
+				if attributeValue != dstPortId {
+					continue EventLoop
+				}
+			case dataTag:
+				rp.packetData = []byte(attributeValue)
+			case toHeightTag:
+				timeout, err := clienttypes.ParseHeight(attributeValue)
+				if err != nil {
+					continue EventLoop
```
Comment (agouin) [inline relayer/provider/cosmos/provider.go:1192]: I changed these from returning the error to continuing on to the next event. I figured we would want to continue attempting to parse other events in the tx (and hold onto potentially prior successfully parsed events) even if a single event fails to parse.

Comment (DavidNix) [reply relayer/provider/cosmos/provider.go:1192]: Log the error there? Swallowing errors in Go has resulted in great pain in my past.

Comment (agouin) [reply relayer/provider/cosmos/provider.go:1192]: good call. added

**Comment by agouin** [comment]
> In this PR or a followup, do we want to include an ibctest/go.mod which points to the latest of ibctest?

Updated

**Pull request merged**